frontend/src/views/MigrationHistoryDetail.vue

```diff
@@ -205,8 +263,21 @@ export default defineComponent({
       );
     });
 
+    const hasBaselineChange = computed(
```

Comment:
isBaselineChange

hasBaselineChange is ambiguous since it could baseline has schema change